assets/js/googlesitekit/datastore/user/permissions.test.js

```diff
@@ -361,8 +384,69 @@ describe( 'core/user authentication', () => {
 					'search-console',
 					'analytics',
 					'pagespeed-insights',
+				] );
+			} );
+
+			it( 'should not include "analytics" module if the dashboardView is GA4', () => {
```

Comment:
Can we change the quotes in these descriptions just to make the formatting consistent throughout these new descriptions and comments? Please can you apply this throughout the PR :)

```suggestion
			it( 'should not include `analytics` module if the dashboardView is GA4', () => {
```

Reply:
Thanks for applying this change. I noticed there were still a couple of cases in this format in `modules.test.js`, so I've pushed an update for this myself to save the round trip :)